src/main/java/com/checkmarx/flow/custom/ServiceNowTracker.java

```diff
@@ -271,10 +274,11 @@ private String createServiceNowTag(ScanRequest request) {
 
     @Override
     public void closeIssue(Issue issue, ScanRequest request) throws MachinaException {
-        log.info("Executing updateIssue Service Now API call");
-        Incident incident = getCloseIncident(request);
```

Comment:
This doesn't seem right, see my comment below on this change on the `getCloseIncident` method changes.